### ISSUE TITLE
Weird grammar and typo in join field docs

### ISSUE THREAD

**Issue by aleph-zero** [elastic/elasticsearch#26590]
**Elasticsearch version** (`bin/elasticsearch --version`): 5.6.0

**Plugins installed**: [] N/A

**JVM version** (`java -version`): N/A

**OS version** (`uname -a` if on a Unix-like system): N/A

**Description of the problem including expected versus actual behavior**:

https://www.elastic.co/guide/en/elasticsearch/reference/5.6/parent-join.html 

This line is hard to parse due to grammar and formatting errors.
```For instance the following index two children documents pointing to the same parent 1 with a `routing value equals to the id of the parent:```

**Comment by jimczi** [comment]
Thanks @aleph-zero
